get-started/part2.md

```diff
@@ -8,76 +8,89 @@ description: Learn how to create a Docker image by writing a Dockerfile, and use
 
-2.  Have a look at the file called `Dockerfile`. Dockerfiles describe how to assemble a private filesystem for a container, and can also contain some metadata describing how to run a container based on this image. The bulletin board app Dockerfile looks like this:
+## Define a container with Dockerfile
 
-    ```dockerfile
-    FROM node:6.11.5    
+Take a look at the file called `Dockerfile` in the bulletin board application. Dockerfiles describe how to assemble a private filesystem for a container, and can also contain some metadata describing how to run a container based on this image. The bulletin board app Dockerfile looks like this:
 
-    WORKDIR /usr/src/app
-    COPY package.json .
-    RUN npm install    
-    COPY . .
+```dockerfile
+# Use the official image as a parent image
+FROM node:6.11.5
 
-    CMD [ "npm", "start" ]    
-    ```
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Copy the file from your host to your current location
+COPY package.json .
+
+# Run the command inside your image filesystem
+RUN npm install
+
+# Copy the rest of your app's source code from your host to your image filesystem.
+COPY . .
 
-    Writing a Dockerfile is the first step to containerizing an application. You can think of these Dockerfile commands as a step-by-step recipe on how to build up our image. This one takes the following steps:
+# Run the specified command within the container.
+CMD [ "npm", "start" ]
+```
 
-    - Start `FROM` the pre-existing `node:6.11.5` image. This is an *official image*, built by the node.js vendors and validated by Docker to be a high-quality image containing the node 6.11.5 interpreter and basic dependencies.
-    - Use `WORKDIR` to specify that all subsequent actions should be taken from the directory `/usr/src/app` *in your image filesystem* (never the host's filesystem).
-    - `COPY` the file `package.json` from your host to the present location (`.`) in your image (so in this case, to `/usr/src/app/package.json`)
-    - `RUN` the command `npm install` inside your image filesystem (which will read `package.json` to determine your app's node dependencies, and install them)
-    - `COPY` in the rest of your app's source code from your host to your image filesystem.
+Writing a Dockerfile is the first step to containerizing an application. You can think of these Dockerfile commands as a step-by-step recipe on how to build up our image. This one takes the following steps:
 
-    You can see that these are much the same steps you might have taken to set up and install your app on your host - but capturing these as a Dockerfile allows us to do the same thing inside a portable, isolated Docker image.
+- Start `FROM` the pre-existing `node:6.11.5` image. This is an *official image*, built by the node.js vendors and validated by Docker to be a high-quality image containing the node 6.11.5 interpreter and basic dependencies.
```

Comment:
Either use `node:current-slim` as image, or at least use an up-to-date version `node:12.14.1`.

The dockersamples PR's now use current-slim as tag which is version 12.14.1 at the time of writing, the current LTS version for Node.js.
The advantage of `current-slim` would be we do not have to update the docs. Pinning to a specific version is best practice though.